packages/paste-website/src/pages/tokens/list/index.mdx

```diff
@@ -4,11 +4,6 @@ description: List of visual attributes (background colors, spacing, etc.) that o
 slug: /tokens/list
 ---
 
-import {graphql} from 'gatsby';
```

Comment:
ℹ️ Moved away from using graphQL to fetch this data, and instead are consuming a build from our tokens package. Check out the PR description for more info.